sphinx/themes/basic/genindex-single.html

```diff
@@ -30,7 +30,7 @@
 {% set title = _('Index') %}
 {% block body %}
 
-<h1 id="index">{% trans key=key %}Index &ndash; {{ key }}{% endtrans %}</h1>
+<h1 id="index">{% trans key=key %}Index &#x2013; {{ key }}{% endtrans %}</h1>
```

Comment:
```suggestion
{# We use ``&#x2013;`` instead of ``&ndash;`` for XHTML compatibility #}
<h1 id="index">{% trans key=key %}Index &#x2013; {{ key }}{% endtrans %}</h1>
```